RPI/co2-monitor.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=CO2 Monitor
+After=network-online.target
+
+[Service]
+ExecStart=/home/pi/.asdf/shims/node main.js
+WorkingDirectory=/home/pi/cs-project
```

Comment:
This script assumes that the content of `RPI` folder is stored inside /home/pi/cs-project